signadot/operator/templates/routeserver-deployment.yaml

```diff
@@ -13,7 +13,9 @@ metadata:
   name: routeserver
   namespace: signadot
 spec:
-  replicas: 1
+{{- with .Values }}{{- with .routeServer }}{{- with .replicas }}
+  replicas: {{ . }}
+{{- else -}}{{- end }}{{- else -}}{{- end }}{{- else -}}{{- end }}
```

Comment:
I am not convinced that this is safe.  Different replicas of routeserver could return different things at different times.  So a client may not be consistent.

We have clients outside of our control.  Also, for a grpc routes watch, it is not clear that the at most one SYNC operation as documented (see https://stackoverflow.com/questions/77993538/is-grpc-connection-always-sticky-when-connecting-to-a-kubernates-service-which-i) and we rely on that.


I would remove the ability to increase the replicas of the routeserver until we figure something out that is known to be consistent for unknown clients and grpc streams with an initial sync operation.